src/app/plugins/register.plugin.ts

```diff
@@ -110,6 +110,12 @@ export class RegisterPlugin extends Plugin {
       invalidClasses
     );
 
+    // Emoji data will be empty if the server has no classes.
+    if (!embedData.emojiData.length) {
+      message.reply('No classes found at this time.');
+      return;
+    }
+
```

Comment:
This isnt obvious why this is true, with the way this is setup
I feel like it would be more obvious if you did `classService.getClasses(ClassType.ALL)` and check if 0